packages/prisma/extensions/disallow-undefined-delete-update-many.ts

```diff
@@ -12,6 +12,10 @@ export const checkUndefinedInValue = (where: any) => {
         throw new Error(message);
       }
 
+      if (whereInput === null) {
+        continue;
+      }
+
```

Comment:
Reason for the issue:-

![Screenshot 2024-11-15 225525](https://github.com/user-attachments/assets/401748a6-8bb0-4ae8-8a0e-8489d9144529)